src/ontology/slurp/omim.tsv

```diff
@@ -1,273 +1,471 @@
-MONDO:0850263	neurodevelopmental disorder with microcephaly, hypotonia, and seizures	OMIM:619876	MONDO:equivalentTo	neurodevelopmental disorder with microcephaly, hypotonia, and seizures		
-MONDO:0850264	dentici-novelli neurodevelopmental syndrome	OMIM:619877	MONDO:equivalentTo	dentici-novelli neurodevelopmental syndrome		
-MONDO:0850265	neurodevelopmental disorder with poor growth and skeletal anomalies	OMIM:619880	MONDO:equivalentTo	neurodevelopmental disorder with poor growth and skeletal anomalies		
-MONDO:0850266	osteoporosis, childhood- or juvenile-onset, with developmental delay	OMIM:619884	MONDO:equivalentTo	osteoporosis, childhood- or juvenile-onset, with developmental delay		
-MONDO:0850267	hepatorenocardiac degenerative fibrosis	OMIM:619902	MONDO:equivalentTo	hepatorenocardiac degenerative fibrosis		
-MONDO:0850268	peripheral motor neuropathy, childhood-onset, biotin-responsive	OMIM:619903	MONDO:equivalentTo	peripheral motor neuropathy, childhood-onset, biotin-responsive		
-MONDO:0850269		OMIM:7	MONDO:equivalentTo			
-MONDO:0850270		OMIM:8	MONDO:equivalentTo			
-MONDO:0850271		OMIM:9	MONDO:equivalentTo			
+MONDO:0850001	acetaminophen metabolism	OMIM:100675	MONDO:equivalentTo	acetaminophen metabolism		
+MONDO:0850002	adeno-associated virus integration site 1	OMIM:102699	MONDO:equivalentTo	adeno-associated virus integration site 1		
+MONDO:0850003	antigen identified by monoclonal antibody 30.2a8	OMIM:107254	MONDO:equivalentTo	antigen identified by monoclonal antibody 30.2a8		
+MONDO:0850004	antigen msk41 identified by monoclonal antibody e3	OMIM:107260	MONDO:equivalentTo	antigen msk41 identified by monoclonal antibody e3		
+MONDO:0850005	serpin peptidase inhibitor, clade a, member 2, pseudogene	OMIM:107410	MONDO:equivalentTo	serpin peptidase inhibitor, clade a, member 2, pseudogene		
+MONDO:0850006	attached cell antigen 28.3.7	OMIM:108990	MONDO:equivalentTo	attached cell antigen 28.3.7		
+MONDO:0850007	autonomously replicating sequence 1	OMIM:109110	MONDO:equivalentTo	autonomously replicating sequence 1		
+MONDO:0850008	baboon m7 virus integration site	OMIM:109180	MONDO:equivalentTo	baboon m7 virus integration site		
+MONDO:0850009	beta-glycerol phosphatase	OMIM:109640	MONDO:equivalentTo	beta-glycerol phosphatase		
+MONDO:0850010	beta-adrenergic stimulation, response to	OMIM:109670	MONDO:equivalentTo	beta-adrenergic stimulation, response to		
+MONDO:0850011	bkm dna	OMIM:109780	MONDO:equivalentTo	bkm DNA		
+MONDO:0850012	blood group--abo suppressor	OMIM:110250	MONDO:equivalentTo	blood group--abo suppressor		
+MONDO:0850013	blood group--abh antigen,  iia 2	OMIM:110310	MONDO:equivalentTo	blood group--abh antigen,  iia 2		
+MONDO:0850014	blood group--en	OMIM:110720	MONDO:equivalentTo	blood group--en		
+MONDO:0850015	blood group--newfoundland	OMIM:111360	MONDO:equivalentTo	blood group--newfoundland		
+MONDO:0850016	blood group, rh system	OMIM:111690	MONDO:equivalentTo	blood group, rh system		
```

Comment:
@joeflack4 could you pls find out where these types of terms come from?

Reply:
Just commenting that I've seen this. If I understand correctly, I believe this is one that we discussed on Mondo call today and that you will be opening a separate issue about this to assign to me.

In particular, the one case I remember us looking at involved something _baboon_ related. FYI I did a simple text search on "baboon" in all the OMIM files, and only a few rows came up. They were in:
- `mimTitles.txt`
- `genemap2.txt`

Reply:
yep, we discussed this. I can open a separate issue.